package.json

```diff
@@ -25,6 +27,8 @@
   },
   "devDependencies": {
     "bl": ">=0.9.0 <0.10.0-0",
+    "coveralls": "^2.11.2",
```

Comment:
@dcousens see my latest rant on the topic starting here: https://twitter.com/rvagg/status/602642984845254656 (follow link in that tweet and read the replies on twitter)